packages/hooks/src/useAuthorize.ts

```diff
@@ -8,7 +8,10 @@ const useAuthorize = (token?: string) => {
     const { accounts, loginid = '' } = client;
     const current_token = accounts[loginid || '']?.token;
 
-    const { data, ...rest } = useFetch('authorize', { payload: { authorize: token || current_token } });
+    const { data, ...rest } = useFetch('authorize', {
+        payload: { authorize: token || current_token },
+        options: { keepPreviousData: true, staleTime: 10000 },
```

Comment:
@sergei-deriv Please help to refactor and fix the issues in the UI instead of patching the data layer 🙏🏻

Reply:
@farzin-deriv agree with UI, I tried to do this yesterday, but found only this solution. Ok, I'll continue to investigate)